terms/src/main/java/dev/ikm/tinkar/terms/TinkarTerm.java

```diff
@@ -2551,6 +2551,15 @@ public class TinkarTerm {
      */
     public static final EntityProxy.Concept FLOAT_FIELD =
             EntityProxy.Concept.make("Float field (SOLOR)", UUID.fromString("6efe7087-3e3c-5b45-8109-90d7652b1506"));
+
+    /**
+     * Java binding for the concept described as <strong><em>Decimal</em></strong>;
+     * identified by UUID: {@code
+     * <a href="http://localhost:8080/terminology/rest/concept/dccb0476-3b63-3d48-b5a2-85bd0ad2fa61">
+     * dccb0476-3b63-3d48-b5a2-85bd0ad2fa61</a>}.
+     */
+    public static final EntityProxy.Concept DECIMAL =
+            EntityProxy.Concept.make("Decimal", UUID.fromString("dccb0476-3b63-3d48-b5a2-85bd0ad2fa61"));
```

Comment:
To be in line with the format of FLOAT, I think the String should be "Decimal (SOLOR)" rather than "Decimal"